<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>jc_Widget</class>
 <widget class="QWidget" name="jc_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>549</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>jc_Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/image/big_background.png);
</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>551</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/image/big_background.png);</string>
   </property>
   <widget class="QPushButton" name="btn_clear">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/CE.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_0">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_leftk">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_rightk">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_backspace">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/del.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/7.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/8.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_multS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>ร</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/4.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_divS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>260</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>330</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/5.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>330</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/6.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-image: url(:/image/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_andS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>๏ผ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>400</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_subS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>400</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Image/3a.png);
border-image: url(:/image/3a.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>20</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/9.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_point">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_equalS">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>470</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>80</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>๏ผ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>381</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>50</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
